modules/openapi-generator/src/main/resources/typescript/api_doc.mustache

```diff
@@ -0,0 +1,84 @@
+# {{moduleName}}.{{classname}}{{#description}}
+
+{{description}}{{/description}}
+
+All URIs are relative to *{{basePath}}*
+
+Method | HTTP request | Description
+------------- | ------------- | -------------
+{{#operations}}{{#operation}}[**{{operationId}}**]({{classname}}.md#{{operationId}}) | **{{httpMethod}}** {{path}} | {{#summary}}{{summary}}{{/summary}}
+{{/operation}}{{/operations}}
+
+{{#operations}}
+{{#operation}}
+# **{{{operationId}}}**
+> {{#returnType}}{{{returnType}}} {{/returnType}}{{{operationId}}}({{#requiredParams}}{{^defaultValue}}{{paramName}}{{^-last}}, {{/-last}}{{/defaultValue}}{{/requiredParams}})
+
+{{#notes}}
+{{{notes}}}
+{{/notes}}
+
+### Example
+
+
+```typescript
+import { {{{moduleName}}} } from '{{{projectName}}}';
+
+const apiInstance = new {{{moduleName}}}.{{classname}}();
+{{#requiredParams}}
+
+// {{{dataType}}}{{#description}} | {{{description}}}{{/description}}
+let {{paramName}} = {{{example}}};
+{{/requiredParams}}
+{{#optionalParams}}
+{{#-first}}
+
+{{/-first}}
+let {{paramName}} = {{{example}}}; // {{{dataType}}} | {{{description}}}{{^required}} (optional){{/required}}{{#defaultValue}} if omitted the server will use the default value of {{{defaultValue}}}{{/defaultValue}}
+{{/optionalParams}}
+{{#hasParams}}
+
+{{/hasParams}}
+apiInstance.{{{operationId}}}({{#allParams}}{{{paramName}}}{{^-last}}, {{/-last}}{{/allParams}}).then((data) => {
+  console.log('API called successfully. Returned data: ' + data);
+}).catch((error) => console.error(error));
+```
+
+
+### Parameters
+{{^allParams}}This endpoint does not need any parameter.{{/allParams}}{{#allParams}}{{#-last}}
+Name | Type | Description  | Notes
+------------- | ------------- | ------------- | -------------{{/-last}}{{/allParams}}
+{{#requiredParams}}{{^defaultValue}} **{{paramName}}** | {{^isPrimitiveType}}**{{{dataType}}}**{{/isPrimitiveType}}{{#isPrimitiveType}}[**{{dataType}}**]{{/isPrimitiveType}}| {{description}} |
```

Comment:
Same here,  you should be aable to browse  `allParams` instead of doing required then optional.

Reply:
Need to remove the `[]` as well.